src/components/common/Map/Map.jsx

```diff
@@ -0,0 +1,25 @@
+import { MapContainer, Marker, Popup, TileLayer } from 'react-leaflet';
+import L from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+
+import marker from 'assets/marker.svg';
+
+const position = [-34.881092955217724, -56.17717361591464];
+
+const myIcon = new L.Icon({
```

Comment:
hey @FlaviaCohen what do you think about renaming this variable with a more meaningful name, right now since I don't know about `leafet` library I'm not aware what is doing this assigment

Reply:
maybe `const MarkerIcon = new Icon({})`

Reply:
I agree!